style.css

```diff
@@ -475,3 +675,15 @@ i {
   background-color: #396df2;
   color: #fff;
 }
+
+@media only screen and (min-width: 768px) {
+  #sec1 > div {
+    width: 65% !important;
+    gap: 10px;
+    top: 10vh;
+  }
+
+  .card > div img {
+    width: 100%;
+  }
```

Comment:
- [ ] Please kindly reduce the size of all the cards images so that we can have a design close to figma design. 😄 

## Figma Design
![png-transparent-nigerian-naira-currency-symbol-symbol-miscellaneous-angle-text](https://user-images.githubusercontent.com/103386183/212946547-50dcef23-a71a-4c15-ba22-0a33fc69ea71.png)

## Your Design
![png-transparent-nigerian-naira-currency-symbol-symbol-miscellaneous-angle-text](https://user-images.githubusercontent.com/103386183/212946929-d17e8772-b709-4d46-91c1-2763b9051140.png)